functions/go/set-namespace/go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/set-na
 
 go 1.17
 
-require github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220316202203-f9115a993ebd
+require github.com/GoogleContainerTools/kpt-functions-sdk/go/fn v0.0.0-20220322205615-b3b2aa33ba46
```

Comment:
Aside: This probably should be a replace to `../fn` (?)  But not really relevant to this PR.

Reply:
Oh whoops - never mind - I missed that this was a different repo!